aiida_aurora/utils/parsers.py

```diff
@@ -71,6 +65,7 @@ def get_data_from_results(array_node):
         'I': I,
         'cn': len(Qd),
         'time-cycles': t[idx[2::2]],
-        'Qd': np.array(Qd),
-        'Qc': np.array(Qc),
+        'Q': cumtrapz(I, t, axis=0, initial=0) / 3.6,
+        'Qd': np.array(Qd) / 3.6,
+        'Qc': np.array(Qc) / 3.6,
```

Comment:
Here have you changed the units of `Q`, `Qd`, and `Qc`?

Reply:
Throughout the app, we treat these charges as capacities. When we do, we generally want them in mAh (ignoring weight normalization for a moment).